### ISSUE TITLE
Stat Panel: auto layout, bottom right data link not clickable

### ISSUE THREAD

**Issue by maemigh** [grafana/grafana#25353]
**What happened**:
Not sure if this should be filed as a bug or not, but using auto-layout where boxes are fairly small, the datalink for the rightmost box on the bottom is not clickable. Instead the resize panel box icon appears.

**What you expected to happen**:
Be able to click on the stat to see the data link. Perhaps the last row could save space at the end for the resize panel icon.

**How to reproduce it (as minimally and precisely as possible)**:
Auto-layout a stat panel with many stats and resize so that each stat is fairly small.

**Anything else we need to know?**:
![Screen Recording 2020-06-04 at 17 18 48 mov](https://user-images.githubusercontent.com/4478053/83733937-3f3e4f80-a689-11ea-9542-5ac64ebd989a.gif)

**Environment**:
- Grafana version: 7.0.3
- Data source type & version: Prometheus
- OS Grafana is installed on: RedHat 7.8
- User OS & Browser: Chrome 81
- Grafana plugins:
- Others:

**Comment by torkelo** [comment]
Strange, thought this code here was handling the collision with window edge: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx#L158

**Comment by peterholmberg** [comment]
@maemigh there's a work around you can do by opening the panel i in view mode. We have to rethink the resize handle and see if this issue gains traction before going forward.

**Comment by mckn** [comment]
I'm closing this one due to inactivity. Please give us a ping if this still is an issue and needs to be re-open.
